src/ext/oblivc/obliv_bits.c

```diff
@@ -899,6 +861,41 @@ void yaoReleaseOt(ProtocolDesc* pd,int me)
   if(me==1) otSenderRelease(&ypd->sender);
   else otRecverRelease(&ypd->recver);
 }
+
+void splitProtocol(ProtocolDesc* pdout, ProtocolDesc * pdin) {
```

Comment:
Point taken. I've clearly not yet gotten the hang of working on projects that other people actually use. My git-fu may not be strong enough yet to make it right retroactively, but I'll give it a try (same comment applies for the other "split into a separate commit" comments, needless to say).